backend/routes/productRoutes.js

```diff
@@ -7,4 +7,20 @@ router.get('/', async (req, res) => {
     res.send(products);
 });
 
+router.post('/', (req, res) => {
+    // I pass the body of the request as argument of the Product constructor and assign to a variable
+    let product = new Product(req.body);
+
+    // I send the product to the db and once the sending is ok I redirect to the same endpoint like before 
+    product.save()
+        .then((result) => {
+            // Another option is to show the resulting item sent to the db; see the line below and uncomment it
+            // res.send(product);
+            res.redirect('/');
```

Comment:
Let's not redirect since the front end is in control of its own stuff. 

Instead can you return the product id from the result?

Reply:
Do you mean a console.log() or a simple return id?

Reply:
do something like `res.send(result.id)`